tests/integration_tests/clouds.py

```diff
@@ -123,8 +123,10 @@ def _get_initial_image(self):
             return image.image_id
 
     def _perform_launch(self, launch_kwargs):
+        wait = launch_kwargs.pop('wait', True)
```

Comment:
If `wait` isn't in `launch_kwargs` then it will default to `True`, meaning that cloud-init failures will raise an exception. (More on this issue further down.)